boiler/src/client/components/Search.jsx

```diff
@@ -1,32 +1,65 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import Store from './Store.jsx';
+import Grid from '@mui/material/Grid';
+import RestaurantEntry from './RestaurantEntry.jsx';
 const key = require('../../../config/keys').yelp.APIkey;
 
 //search restaurants
 
 const Search = () => {
 
-  const [ store, useStore ] = useState([]);
-
-  const getCrawfish = () => {
-    return axios.get('https://cors-anywhere.herokuapp.com/https://api.yelp.com/v3/businesses/search?location=neworleans&categories=crawfish', {
+  // const [ store, useStore ] = useState([]);
+  
+  const [restaurants, setRestaurants] = useState([]);
+  
+  
+  const getRestaurants = () => {
+    axios.get('/api/restaurants')
+    .then((results) => {
+      console.log('results', results.data);
```

Comment:
once you get it working, remove console logs before pushing to github (don't clutter your teammates' consoles)